### PR TITLE
[CTOR-612] refresh Informix DB SNMP doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 1]
```diff
@@ -96,102 +131,417 @@ sur le serveur comme indiqué sur la documentation officielle :
...
 
-Tous les modes disponibles peuvent être affichés en ajoutant le paramètre 
-`--list-mode` à la commande:
+La commande devrait retourner un message de sortie similaire à :
 
 ```bash
-/usr/lib/centreon/plugins//centreon_informix_snmp.pl \
-    --plugin=database::informix::snmp::plugin \
-    --list-mode
+OK: All instances are ok | '*global*#sessions'=;;;0;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:345]: ;;;0;?

File: pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 2]
```diff
@@ -5,80 +5,114 @@ title: Informix DB SNMP
...
+<Tabs groupId="sync">
+<TabItem value="App-DB-Informix-SNMP-custom" label="App-DB-Informix-SNMP-custom">
 
-It brings the following Service Templates:
+| Service Alias | Service Template                          | Service Description     |
+|:--------------|:------------------------------------------|:------------------------|
+| Archivelevel0 | App-DB-Informix-Archivelevel0-SNMP-custom | Check last full backup  |
+| Chunk-Status  | App-DB-Informix-Chunk-Status-SNMP-custom  | Check chunk status      |
+| Dbspace-Usage | App-DB-Informix-Dbspace-Usage-SNMP-custom | Check dbspaces usage    |
+| Global-Cache  | App-DB-Informix-Global-Cache-SNMP-custom  | Check write/read cached |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:26]:
```suggestion
| Global-Cache  | App-DB-Informix-Global-Cache-SNMP-custom  | Check write/read cache |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 3]
```diff
@@ -5,80 +5,114 @@ title: Informix DB SNMP
...
+| Lock-Stats    | App-DB-Informix-Lock-Stats-SNMP-custom    | Check instance locks    |
+| Logfile-Usage | App-DB-Informix-Logfile-Usage-SNMP-custom | Check log files usage   |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:28]:
```suggestion
| Logfile-Usage | App-DB-Informix-Logfile-Usage-SNMP-custom | Check log file usage   |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 4]
```diff
@@ -88,109 +122,425 @@ It brings the following Service Templates:
...
+	--snmp-community='my-snmp-community'  \
+	--warning-sessions='' \
+	--critical-sessions=''
 ```
 
 The expected command output is shown below:
 
 ```bash
-OK: 7 client sessions | sessions=7;10;20;0;;
+OK: All instances are ok | '*global*#sessions'=;;;0;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:344]: same as FR

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 5]
```diff
@@ -5,80 +5,115 @@ title: Informix DB SNMP
...
-| Lock-Stats    | App-DB-Informix-Lock-Stats-SNMP    | X       |
-| Logfile-Usage | App-DB-Informix-Logfile-Usage-SNMP | X       |
-| Sessions      | App-DB-Informix-Sessions-SNMP      | X       |
+### Règles de découverte
+
+#### Découverte d'hôtes
+
+| Nom de la règle | Description                                                                                                                                                                                                                                         |
+|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **App-DB-Informix-SNMP-custom** host template |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:43]:
```suggestion
| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](https://docs.centreon.com/fr/pp/integrations/plugin-packs/procedures/applications-protocol-snmp/) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **App-DB-Informix-SNMP-custom** |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 6]
```diff
@@ -96,102 +131,417 @@ sur le serveur comme indiqué sur la documentation officielle :
...
+|:---------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------|:-----------:|
+| FILTERNAME     | Filter chunk name (can be a regexp)                                                                                                                               |                             |             |
+| UNKNOWNSTATUS  | Define the conditions to match for the status to be UNKNOWN (default: ''). You can use the following variables: %{status}, %{display}                             |                             |             |
+| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%{status} =~ /inconsistent/'). You can use the following variables: %{status}, %{display} | %{status} =~ /inconsistent/ |             |
+| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING (default: ''). You can use the following variables: %{status}, %{display}                             |                             |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:260]:
```suggestion
| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING. You can use the following variables: %{status}, %{display}                             |                             |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 7]
```diff
@@ -96,102 +131,417 @@ sur le serveur comme indiqué sur la documentation officielle :
...
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                   |             |
+
+</TabItem>
+<TabItem value="Chunk-Status" label="Chunk-Status">
+
+| Macro          | Description                                                                                                                                                       | Valeur par défaut           | Obligatoire |
+|:---------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------|:-----------:|
+| FILTERNAME     | Filter chunk name (can be a regexp)                                                                                                                               |                             |             |
+| UNKNOWNSTATUS  | Define the conditions to match for the status to be UNKNOWN (default: ''). You can use the following variables: %{status}, %{display}                             |                             |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:258]:
```suggestion
| UNKNOWNSTATUS  | Define the conditions to match for the status to be UNKNOWN. You can use the following variables: %{status}, %{display}                             |                             |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 8]
```diff
@@ -88,109 +122,425 @@ It brings the following Service Templates:
...
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). |                   |             |
+
+</TabItem>
+<TabItem value="Chunk-Status" label="Chunk-Status">
+
+| Macro          | Description                                                                                                                                                       | Default value               | Mandatory   |
+|:---------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------|:-----------:|
+| FILTERNAME     | Filter chunk name (can be a regexp)                                                                                                                               |                             |             |
+| UNKNOWNSTATUS  | Define the conditions to match for the status to be UNKNOWN (default: ''). You can use the following variables: %{status}, %{display}                             |                             |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:259]:
```suggestion
| UNKNOWNSTATUS  | Define the conditions to match for the status to be UNKNOWN. You can use the following variables: %{status}, %{display}                             |                             |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 9]
```diff
@@ -88,109 +122,425 @@ It brings the following Service Templates:
...
+| CRITICALSTATUS | Define the conditions to match for the status to be CRITICAL (default: '%{status} =~ /inconsistent/'). You can use the following variables: %{status}, %{display} | %{status} =~ /inconsistent/ |             |
+| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING (default: ''). You can use the following variables: %{status}, %{display}                             |                             |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:261]:
```suggestion
| WARNINGSTATUS  | Define the conditions to match for the status to be WARNING. You can use the following variables: %{status}, %{display}                             |                             |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 10]
```diff
@@ -88,109 +122,425 @@ It brings the following Service Templates:
...
-    --critical-sessions='20' \
-    --use-new-perfdata 
+/usr/lib/centreon/plugins/centreon_informix_snmp.pl \
+	--plugin=database::informix::snmp::plugin \
+	--mode=sessions \
+	--hostname=10.0.0.1 \
+	--snmp-version='2c' \
+	--snmp-community='my-snmp-community'  \
+	--warning-sessions='' \
+	--critical-sessions=''
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:338]:
```suggestion
	--warning-sessions='10' \
	--critical-sessions='20'
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md [hunk 11]
```diff
@@ -96,102 +131,417 @@ sur le serveur comme indiqué sur la documentation officielle :
...
-    --mode=sessions \
-    --help
+/usr/lib/centreon/plugins/centreon_informix_snmp.pl \
+	--plugin=database::informix::snmp::plugin \
+	--mode=sessions \
+	--hostname=10.0.0.1 \
+	--snmp-version='2c' \
+	--snmp-community='my-snmp-community'  \
+	--warning-sessions='' \
+	--critical-sessions=''
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-informix-snmp.md:339]:
```suggestion
	--warning-sessions='10' \
	--critical-sessions='20'
```

**Review by lucie-dubrunfaut — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**